comparison/loadtest.go

```diff
@@ -157,12 +157,20 @@ func initLoadTest(t *terraform.Terraform, buildCfg BuildConfig, dumpFilename str
 		clients: appClients,
 	}
 
-	binaryPath := "/opt/mattermost/bin/mattermost"
 	resetCmd := cmd{
 		msg:     "Resetting database",
-		value:   fmt.Sprintf("%s reset --confirm", binaryPath),
 		clients: []*ssh.Client{appClients[0]},
 	}
+	switch dpConfig.TerraformDBSettings.InstanceEngine {
+	case "aurora-postgresql":
+		subCmd := fmt.Sprintf("-U %s -h %s %sdb", dpConfig.TerraformDBSettings.UserName, tfOutput.DBCluster.ClusterEndpoint, dpConfig.ClusterName)
```

Comment:
Should we refactor this logic of "ClusterName"+db out?

Reply:
Seems reasonable :+1: